superset-frontend/jest.config.js

```diff
@@ -16,33 +16,64 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 module.exports = {
-  testRegex: '(\\/spec|\\/src)\\/.*(_spec|\\.test)\\.(j|t)sx?$',
+  testRegex: '\\/(spec|src|plugins|packages)\\/.*(_spec|\\.test)\\.[jt]sx?$',
+  testPathIgnorePatterns: [
+    'packages\\/generator-superset\\/.*',
```

Comment:
should we add a TODO here to figure out why this was crashing tests?

Reply:
I added a step in the [workflow](https://github.com/apache/superset/pull/17552/commits/fcbcca61b6662f9e63161e1184371392c489ef38) of frontend for this unit test.